src/models.py

```diff
@@ -51,22 +52,44 @@ def create_session(self) -> UserSession:
         session.commit()
         return new_session
 
+    @staticmethod
+    def get_users() -> list[Any]:
```

Comment:
To take advantage of the typing system we should annotate types as strict as possible. This function will always return list of `User` objects, so we can annotate it as `list[User]`.

And we can do the same thing for `get_topics()` method